### PR TITLE
front: nge language synchro + Remove SBB header + bump NGE version 2.9.9

### LINKED ISSUE #9694: front: nge disable SBB header + language synchronization
# Description

Parent issue : https://github.com/osrd-project/osrd-confidential/issues/855

Also close the enhancement #8569 

- [x] [Allow language synchronization in NGE](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/pull/349)
- [x] [Remove SBB header from fork](https://github.com/osrd-project/netzgrafik-editor-frontend/pull/4)
- [x] Add credits for SBB
- [x] Bump NGE package version in OSRD

### PR DESCRIPTION
**Pull request by louisgreiner**
Close https://github.com/OpenRailAssociation/osrd/issues/9694

- Allow language synchronization
- Remove sbb header
- Credit sbb
- bump forked NGE version to 2.9.9

image

note to myself : remember to squash before merge

### CODE REVIEW

File: front/src/common/ReleaseInformations/LicenseAttributions.tsx
Comment (Uriel-Sautron) [inline front/src/common/ReleaseInformations/LicenseAttributions.tsx]:
```suggestion
      <h2 className="text-center my-4">{t('informations.dataSources')}</h2>
```

**Review by Uriel-Sautron — APPROVED** [review]
LGTM, tested just a comment

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, good job !

Maybe we can wait for @emersion or @flomonster approval for the license part to merge :)

File: front/src/common/ReleaseInformations/LicenseAttributions.tsx
Comment (emersion) [inline front/src/common/ReleaseInformations/LicenseAttributions.tsx]: Nit: can we leave out the version? It will be cumbersome to keep in sync manually.

File: front/src/common/ReleaseInformations/LicenseAttributions.tsx
Comment (emersion) [inline front/src/common/ReleaseInformations/LicenseAttributions.tsx]: This is supposed to be human-readable text I think? So "Apache-2.0" would be more appropriate I think?

**Review by emersion — APPROVED** [review]
LGTM in any case

**Pull request merged**
